tests/test_formatting.py

```diff
@@ -79,6 +79,42 @@ def test_format_comment_separator(self):
             expected.splitlines(),
         )
 
+    def test_init_lists(self):
+        # Square brackets should have no spaces after [ and before ]
+        self.assertEqual(self._format(b"[ ];").rstrip(), b"[];")
+        self.assertEqual(self._format(b"[ 1 ];").rstrip(), b"[1];")
+        self.assertEqual(self._format(b"[ 1 , 2 ];").rstrip(), b"[1, 2];")
```

Comment:
While we are at it let's also add something testing trailing `,`, e.g.,

```suggestion
        self.assertEqual(self._format(b"[ 1 ];").rstrip(), b"[1];")
        self.assertEqual(self._format(b"[ 1 ,];").rstrip(), b"[1, ];")
        self.assertEqual(self._format(b"[ 1 , 2 ];").rstrip(), b"[1, 2];")
        self.assertEqual(self._format(b"[ 1 , 2 , ];").rstrip(), b"[1, 2, ];")
```